Updated sdk automation docs

raych1 opened the pull request:
- updated specs
- updated user guidance pages

raych1 committed:
Updated sdk automation docs

weshaggard reviewed (approved):
Did a quick scan and it seems reasonable. Do you know if anyone consumes these? Should the schemas actually go next to the tools and potentially be used to test the inputs and outputs?

raych1 committed:
Deprecated contracts docs and refereces to original ones in source

raych1 commented:
> Yes I would remove these and link to the ones in source.

I removed these docs from spec repo and changed the references to the original source files.

maririos commented on documentation/sdkautomation/README.md:
```diff
@@ -1,22 +1,22 @@
+2. Identify the TypeSpec project or the `readme.md` file based on the PR changed files.
 
-3. Filter which sdk will be generated:
+3. Trigger the `spec-gen-sdk` pipelines for five SDK languages.
```
it would depend on the languages specified for the project, no?

maririos replied:
Answered my question below :)

raych1 replied:
No, it will always trigger five pipelines. Within the pipeline run, it will report the warnings if the specific language is not enabled the SDK configuration.
